ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -23,8 +23,8 @@ struct CreateTaskView: View {
                         .frame(minHeight: 44)
```

Comment:
<img width="269" alt="image" src="https://user-images.githubusercontent.com/24592433/205722716-42a7d9e9-4d73-4470-ae0e-5df44af56a09.png">
this colours expected to be primary text
<img width="269" alt="image" src="https://user-images.githubusercontent.com/24592433/205722789-62e63125-c31d-447f-9b50-f75801e821a4.png">
calendar picker colour expected to be secondaryAccent